### PR TITLE
delete skin file about "*.fnt" and use nativefont instead.

### PR DESCRIPTION
**Pull request by lishaoxia1985**
1. skin's default font should be english font. if not, using chinese font directly will lead to that some font misses in screen. i think this bug is hard to solve. 
2. now we can make the file "skin/flat-earth-ui.png" smaller. but i don't know which tools can be used to edit it.

### CODE REVIEW

**Comment by yairm210** [comment]
Apart from deleting a file that is no longer in use, what do we gain from adding the font to the skin?

File: core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt [hunk 1]
```diff
@@ -71,19 +73,22 @@ open class CameraStageBaseScreen : Screen {
     }
 
     companion object {
-        var skin = Skin(Gdx.files.internal("skin/flat-earth-ui.json"))
+        var skin = Skin().apply {
+            add("Nativefont", Fonts().getFont(45, true), BitmapFont::class.java)
```
Comment (lishaoxia1985) [inline core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: you can see this code.

Comment (yairm210) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: Yes, but what does this actually change? In what way does it improve the game to have this as part of the skin?

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: if we use this flat-earth-ui.png don't need to contain font picture.
you can see this:
 https://stackoverflow.com/questions/24856201/putting-freetypefont-into-libgdx-skin
i also edit bitmapfont in file "android/assets/skin/flat-earth-ui.json".

Comment (yairm210) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: You're missing the point...
I get why to remove the existing font from the skin, I don't get why we need to add our font to the skin. We can just not add the NativeFont to the skin at all.

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: skin must include bitmapfont,  you can look at 
 "android/assets/skin/flat-earth-ui.json", 
for example, com.badlogic.gdx.scenes.scene2d.ui.TextButton$TextButtonStyle has a key "font", if we don't add the NativeFont to the skin, we should not delete "*.fnt". if not, bitmapfont will be null.

Comment (yairm210) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: I just realized that this means that the getFonts() will occur upon creating the first CameraStageBaseScreen, which means - upon creating the loading screen, and not "while loading", which could lead to many ANRs, as there were before the loading screen existed 0_0

Comment (lishaoxia1985) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: yes, you get it.😃

Comment (yairm210) [reply core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt]: This is Not Good =/
If that's what this means, then this is not a change that I want to occur

**Comment by vainiovano** [comment]
So, is this change ready now?

**Comment by yairm210** [comment]
Yes!
Since creating the font NO LONGER creates all associated char glyphs, then lazily creating the skin for the loading screen won't cause ANR problems!

**Pull request merged**